requirements-tests.txt

```diff
@@ -7,7 +7,7 @@ mypy ==1.3.0
 ruff ==0.0.272
 black == 23.3.0
 httpx >=0.23.0,<0.24.0
-email_validator >=2.0.0,<3.0.0
+email_validator >=2.0.0.post2,<3.0.0
```

Comment:
Is there any reason why this change was needed?

Reply:
With the environment, installed by default, tests not pass. So, I update email-validator and it fixes a some of them.

Reply:
Strange, I just manually installed explicitly `2.0.0` to see what was breaking 🤔 

Let's see if reverting works or if I have to put it back.